### DISCUSSION TITLE
ConfigProvider does not apply the locale

### DISCUSSION THREAD

**Discussion by yantakus** [ant-design/ant-design#37211] [Q&A] [answered]
Hi there. I'm not sure where to put it best, but I've created an issue a few days ago. It wasn't resolved and closed. Could someone please have a look. Here it is: https://github.com/ant-design/ant-design/issues/37184

**Comment by zombieJ** [comment, marked as answer]
https://github.com/ant-design/ant-design/issues/37184#issuecomment-1225295982

**Comment by quick691fr** [comment]
I got the same problem with latest 5.24.3 version using default dayjs with `import "dayjs/locale/fr"`. 
Some part of the component are correctly transated but not the weekdays and monthdays (see screenshot).
Any idea of where it should come from ?
![image](https://github.com/user-attachments/assets/fadc90de-3db9-465f-8715-e9d73910365c)

**Reply by quick691fr** [reply]
@zombieJ  ?

**Reply by quick691fr** [reply]
Forget it, was an included package which override the dayjs version to older one...
